core/preloop.gms

```diff
@@ -38,13 +38,13 @@ vm_co2eqMkt.l(ttot,regi,emiMkt) = 0;
 v_shfe.l(t,regi,enty,sector) = 0;
 v_shGasLiq_fe.l(t,regi,sector) = 0;  
 pm_share_CCS_CCO2(t,regi) = 0; 
-  
+pm_pvpRegi(ttot,regi,enty) = 0;
```

Comment:
shouldn't this be only for t and not ttot?
so that if the run is fixed to another run, it will have the pm_pvpRegi values for the fixed time steps?

ah, likely I am wrong - this will anyway happen later in the read-in of input_ref, right?

Reply:
this will be addressed in 
(cm_startyear gt 2005),
  Execute_Loadpoint 'input_ref' p_pvpRegiBeforeStartYear = pm_pvpRegi;
   pm_pvpRegi(ttot,regi,"perm")$((ttot.val gt 2005) AND (ttot.val lt cm_startyear)) =  p_pvpRegiBeforeStartYear(ttot,regi,"perm");
 );